src/DssSpell.sol

```diff
@@ -21,23 +21,101 @@ pragma solidity 0.6.12;
-    function ilks(bytes32) external view returns (string memory, address, uint48, uint48);
-    function init(bytes32, uint256, string calldata, uint48) external;
-    function bump(bytes32 ilk, uint256 val) external;
+import { VatAbstract } from "dss-interfaces/dss/VatAbstract.sol";
+import { JugAbstract } from "dss-interfaces/dss/JugAbstract.sol";
+import { IlkRegistryAbstract } from "dss-interfaces/dss/IlkRegistryAbstract.sol";
+
+interface D3MHubLike {
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+}
+
+interface D3MCompoundPoolLike {
+    function hub() external view returns (address);
+    function ilk() external view returns (bytes32);
+    function vat() external view returns (address);
+    function comptroller() external view returns (address);
+    function comp() external view returns (address);
+    function dai() external view returns (address);
+    function cDai() external view returns (address);
+}
+
+interface D3MCompoundPlanLike {
+    function tack() external view returns (address);
+    function delegate() external view returns (address);
+    function cDai() external view returns (address);
+}
+
+interface D3MOracleLike {
+    function vat() external view returns (address);
+    function ilk() external view returns (bytes32);
+}
+
+interface CDaiLike {
+    function interestRateModel() external view returns (address);
+    function implementation() external view returns (address);
+}
+
+interface StarknetGovRelayLike {
+    function relay(uint256 spell) external;
 }
 
-interface ChainlogLike {
-    function removeAddress(bytes32) external;
+interface OSMLike {
+    function src() external view returns (address);
+}
+
+interface OracleLiftLike {
+    function lift(address[] calldata) external;
+}
+
+interface TokenLike {
+    function transfer(address, uint256) external returns (bool);
+    function decimals() external view returns (uint8);
+    function name() external view returns (string memory);
+    function symbol() external view returns (string memory);
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d8c711404e9e17db24b611aa03d45507c1993148/governance/votes/Executive%20vote%20-%20November%2016%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/c94660aff7fa774aacb5c78ee7068ef4b5d9d243/governance/votes/Executive%20vote%20-%20December%202%2C%202022.md -q -O - 2>/dev/null)"
 
     string public constant override description =
-        "2022-11-16 MakerDAO Executive Spell | Hash: 0xf3023b0a9b3c423edf28bf4cdbd61d287ab29265d462a5fae0378227c153ea7b";
+        "2022-11-30 MakerDAO Executive Spell | Hash: 0xd2c190b0173eec16d7039cae64ce076e7abb18572af5d5a7fcb9b1b1c30bf170";
```

Comment:
date should be updated

Reply:
Do we normally put the updated date? I thought it was set to when it's supposed to go out originally.

Reply:
`2022-12-02` is specified in the exec copy

Reply:
fixed.